editor/plugins/org.fusesource.ide.wsdl2rest.ui/src/org/fusesource/ide/wsdl2rest/ui/wizard/Wsdl2RestWizard.java

```diff
@@ -283,8 +291,8 @@ private IResource getFileForPath(IProject project, IPath path, IProgressMonitor
 	 * @param monitor
 	 * @throws Exception
 	 */
-	private void generate(IProgressMonitor monitor) throws Exception {
-		SubMonitor subMon = SubMonitor.convert(monitor, 5);
+	private void generate(IProgressMonitor monitor) throws IOException, CoreException, URISyntaxException {
+		SubMonitor subMon = SubMonitor.convert(monitor, 6);
```

Comment:
I think that it should stay to 5, can you double-check?

Reply:
If you look at line 310 you will find a .split(2). So 6 is correct.